src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,81 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  onChange: React.Dispatch<React.SetStateAction<Todo[]>>;
+  visibleTodos: Todo[];
+};
+
+function selectTodos(selectedOption: string, todo: Todo) {
+  if (selectedOption === 'active') {
+    return todo.completed === false;
+  }
+
+  if (selectedOption === 'completed') {
+    return todo.completed === true;
+  }
+
+  return true;
```

Comment:
use switch... case, also
```suggestion
  if (selectedOption === 'active') {
    return !todo.completed;
  }

  if (selectedOption === 'completed') {
    return todo.completed;
  }

  return true;
```

Reply:
it's already boolean values, so not needed to convert